sway-core/src/semantic_analysis/ast_node/declaration/storage.rs

```diff
@@ -58,6 +59,7 @@ impl ty::TyStorageField {
         md_mgr: &mut MetadataManager,
         module: Module,
         ix: &StateIndex,
+        namespace: Option<&str>,
```

Comment:
While I am at nit picking :smile: here as well, `namespace` first (or `ns` to be fully consistent?).